### PR TITLE
TRIVIAL Update codeowners file

### PR DESCRIPTION
**Pull request by nerkis**

### CODE REVIEW

File: docs/CODEOWNERS [hunk 1]
```diff
@@ -1,7 +1,2 @@
-# -*- mode: gitignore; -*-
```
Comment (nerkis) [inline docs/CODEOWNERS:1]: @MXWest I hadn't seen this before. Did you want it here on purpose?

Comment (MXWest) [reply docs/CODEOWNERS:1]: I honestly don't know what that is.

Comment (MXWest) [reply docs/CODEOWNERS:1]: I think it's there for code editors to understand that CODEOWNERS has the same syntax as `gitignore` for code highlighting.

**Review by MXWest — APPROVED** [review]
🌸

**Pull request merged**